src/app/story/story.component.ts

```diff
@@ -20,6 +21,9 @@ export class StoryComponent implements OnInit, AfterViewInit, OnDestroy {
   loading: boolean;
   story: StoryAPI;
   selectedNode: any;
+  totalLeadTime: string;
```

Comment:
That's my ignorance... but what does "lead" mean in this case?

Reply:
It means total time it takes to get through the pipeline. This was the name Ralph used in the MVP, so just wanted to keep it consistent :smile:

Reply:
Do you think it'd be helpful to leave a comment here explaining that?